main.js

```diff
@@ -15,5 +18,22 @@ document.addEventListener('keydown', function (event) {
   } else if (event.key === 'ArrowLeft') {
     carData.direction = 'west';
   }
+
   $car.setAttribute('class', 'car ' + carData.direction);
+
+  if (event.key === ' ') {
+
+    leftValue = $car.style.left;
+    pIndex = leftValue.indexOf('p');
+
+    leftValue.substring(0, pIndex);
+    var leftNumber = Number(leftValue);
+
+    setInterval(function () {
+      leftNumber += 3;
```

Comment:
This would be much more simple if you just used an x property that started off with 0, and then modified it in the interval, and then updated the object with the new position on the pag.e